### PR TITLE
Ensure subject age is int

### LINKED ISSUE #773: Subject age sometimes a float
On our dev site:

```
>>> sessions = fw.GetAllSessions()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "flywheel.py", line 49, in GetAllSessions
    return self._handle_return(status, payload)
  File "flywheel.py", line 38, in _handle_return
    raise FlywheelException(result)
flywheel.FlywheelException: json: cannot unmarshal number 1009152000.0 into Go struct field Subject.age of type int
```

From our [data model](https://github.com/scitran/core/wiki/Data-Model):

> age 	int 	age at time of session in seconds

### PR DESCRIPTION
**Pull request by nagem**
Fixes #773 
Addresses some of #777 

@kofalt, @gsfr
DB update and json schema change. Also added a few tests. 

@ryansanford 
While I was in `database.py` I added a few quality of life changes to logging. I would like to add more progress information to `process_cursor` since that's likely where you were experiencing the problem, but didn't want testing work there to hold up this PR. 

### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md

### CODE REVIEW

**Comment by nagem** [comment]
This was a simple change that has tests, going to merge now...

**Comment by gsfr** [comment]
Awesome, @nagem. Thanks for the quick turnaround.

**Comment by ryansanford** [comment]
@nagem Does this need to change to integer as well? https://github.com/scitran/core/blob/10a61b6d7b095521ef6225445617424edfac5c26/raml/schemas/mongo/subject.json#L11

**Comment by nagem** [comment]
@ryansanford It should for consistency, but it's really just stopping us from shooting ourselves in the foot after sanitizing user input. We rarely use the mongo schemas, anytime we do is just lingering code. 

I'll make an update there, too - thanks for catching that.

File: bin/database.py [hunk 1]
```diff
@@ -1003,7 +1019,9 @@ def upgrade_schema():
         while db_version < CURRENT_DATABASE_VERSION:
             db_version += 1
             upgrade_script = 'upgrade_to_'+str(db_version)
+            logging.info('Upgrading to version {} ...'.format(db_version))
             globals()[upgrade_script]()
+            logging.info('Upgrade to version {} complete.'.format(db_version))
```
Comment (kofalt) [inline bin/database.py]: Nice, I like this adding information here.

File: bin/database.py [hunk 2]
```diff
@@ -93,6 +93,7 @@ def process_cursor(cursor, closure):
     # pool.close()
     # pool.join()
 
+    logging.info('Proccessing {} items in cursor ...'.format(cursor.count()))
```
Comment (nagem) [inline bin/database.py]: @kofalt I was planning on adding "`n` of `count` items processed..." somewhere around here. Maybe divide it into 10 or 20 groups. Just didn't want to make it part of this PR.

Comment (kofalt) [inline bin/database.py]: Totally, I wouldn't add anything unless it's urgent.

**Pull request merged**